contracts/src/v0.8/functions/dev/1_0_0/libraries/FunctionsRequest.sol

```diff
@@ -27,6 +27,7 @@ library FunctionsRequest {
     CodeLanguage language; // The coding language that the source code is written in
     string source; // Raw source code for Request.codeLocation of Location.Inline, URL for Request.codeLocation of Location.Remote, or slot decimal number for Request.codeLocation of Location.DONHosted
     bytes encryptedSecretsReference; // Encrypted URLs for Request.secretsLocation of Location.Remote (use addSecretsReference()), or CBOR encoded slotid+version for Request.secretsLocation of Location.DONHosted (use addDONHostedSecrets())
+    bytes requestSignature; // Signature generated by the subscription owner's EOA
```

Comment:
Solidity wise this looks good. Not sure on how it will be used but no blockers from me

Reply:
I am creating a follow-up PR which will use this properly.  I wanted to get in everything I had before the weekend started.  I am going to try to get the PR which uses this (and removes it from being optional) tonight.